bundles/org.openhab.voice.googletts/src/main/java/org/openhab/voice/googletts/internal/GoogleTTSService.java

```diff
@@ -361,4 +347,19 @@ private AudioFormat parseAudioFormat(byte[] audio) throws TTSException {
             throw new TTSException("Cannot parse WAV format", e);
         }
     }
+
+    @Override
+    public @NonNull String getCacheKey(@NonNull String text, @NonNull Voice voice,
+            @NonNull AudioFormat requestedFormat) {
+        try {
+            MessageDigest md = MessageDigest.getInstance("MD5");
+            byte[] bytesOfMessage = (config.toConfigString() + text + requestedFormat).getBytes(StandardCharsets.UTF_8);
+            String hash = String.format("%032x", new BigInteger(1, md.digest(bytesOfMessage)));
+            return ((GoogleTTSVoice) voice).getTechnicalName() + "_" + hash;
+        } catch (NoSuchAlgorithmException e) {
+            // should not happen
+            logger.error("Could not create MD5 hash for '{}'", text, e);
+            return "nomd5algorithm";
```

Comment:
A WARN level should be better.
I am asking myself if you could not just call the super class if your method fails ?

Reply:
I already read your comment in the other PR, so forget my remark about calling the super class.

Please just change the log level.

Reply:
@dalgwen : please consider changing the log level and then I will merge.

Reply:
Done